### ISSUE TITLE
add rai init

### ISSUE THREAD

**Issue by abduld** [rai-project/rai#16]
(Thanks for reporting an issue! Please make sure you click the link above to read contributor guidelines, then fill out the details below)

## What are the minimum necessary steps to reproduce this issue?

1. …
2. …
3. …

## What happens?

…

## What were you expecting to happen?

…

## Please paste any error or log messages here

(if long, you can [link to a gist](https://gist.github.com/))

## Any other details

…

**Comment by cwpearson** [comment]
Is this supposed to be a client command that makes a template of a rai_build.yml and a template .rai_profile, or something?

**Comment by abduld** [comment]
it would make the `.rai_profile` by asking a series of questions (or you just paste the content of the email). it's meant to avoid dealing with what the homedirector for windows is, where the .rai_profile should be placed, etc...

**Comment by JinjunXiong** [comment]
The .rai_profile is still needed to save the credentials (we do want to have some control on who can acces RAID by letting users obtain credentials from RAI sys admin). A simple solution would be to have .rai_proifle  always present in the current project directory, but not to be added into the zipped build directory for download (as this may create potential risk if someone else happens to downloaded that zipped build file - the credentials may be leaked).

**Comment by cwpearson** [comment]
Right, I think @abduld is saying that this would be a little subcommand that would help someone get their `.rai_profile` in the right place and with the right format. When we use rai for the courses, a few people always have problems getting `.rai_profile` in their home directory. This would not give people a rai secret, they would still need to get that from an admin.
